src/LenderNft.hs

```diff
@@ -30,53 +30,47 @@ import qualified Ledger.Typed.Scripts     as Scripts
 import           Ledger.Value             as Value
 import qualified PlutusTx
 import           PlutusTx.Prelude         hiding (Semigroup (..), unless)
-import qualified Plutus.V1.Ledger.Scripts as Plutus
-import           Prelude                  (Semigroup (..), Show)
 import qualified Common.Utils             as U
 
-{-# INLINABLE lender #-}
-lender :: TokenName
-lender = TokenName { unTokenName = consByteString 76 emptyByteString }  -- L
-
 {-# INLINABLE mkPolicy #-}
-mkPolicy :: ValidatorHash -> TxOutRef -> Integer -> ScriptContext -> Bool
-mkPolicy vh utxo _ ctx = validate
+mkPolicy :: TxOutRef -> ScriptContext -> Bool
+mkPolicy utxo ctx = all validate mintedValue
```

Comment:
It's not possible to mint multiple such NFTs inside the same transaction and that's a good thing as it prevents minting multiple NFTs with the same token name. However, if that's the case, you don't need to use `all` and straight away assert that the `mintedValue` would contain exactly one triplet (maybe after filtering for `amount  > 0`, if you want).

Reply:
Implemented change, like it is done in [lobster challenge](https://github.com/input-output-hk/lobster-challenge/blob/9d349f94400460de99fb4fe4d7dbed3585a264be/src/Cardano/PlutusLobster/LobsterScript.hs#L64)